spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java

```diff
@@ -69,6 +71,9 @@
 	@Autowired
 	private ConfigurableApplicationContext context;
 
+	@Autowired
```

Comment:
no - I'm just saying that the field could be null if no `DelegatingResourceLoader` is defined in the context... in that case, a `DefaultResourceLoader` could be used directly